src/Symfony/Bridge/Doctrine/DependencyInjection/CompilerPass/DoctrineValidationPass.php

```diff
@@ -43,12 +40,8 @@ public function process(ContainerBuilder $container)
     /**
      * Gets the validation mapping files for the format and extends them with
      * files matching a doctrine search pattern (Resources/config/validation.orm.xml).
-     *
-     * @param ContainerBuilder $container
-     * @param string           $mapping
-     * @param string           $extension
      */
-    private function updateValidatorMappingFiles(ContainerBuilder $container, $mapping, $extension)
+    private function updateValidatorMappingFiles(ContainerBuilder $container, string $mapping, string $extension)
```

Comment:
shouldn't this have void return typehint?

Reply:
I find void useless, so I didn't add any.
If you want to make it your battle, please do :)

Reply:
Method without void can mean two types of things - either it's return type void, or it's just not filled in or the actual return type is in DocBlock. That's a lot of "or"s :)

For me, it would make sense to add it if it were to be enforced by phpcs (method either has return type hint, or `@return` docblock). Is there a plan to enforce return typehints?

Reply:
no plan I know about

Reply:
https://github.com/symfony/symfony/issues/24882